python/uperf/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/jaredoconnell/arcaflow-plugin-sdk-python.git@5c2623bc64b1da753de06b626ecbc267b7b69b62
```

Comment:
this needs to be changed before merge

Reply:
Correct. I need to wait for my arcaflow-plugin-sdk-python changes to be merged in for that. It's not compatible with the releases.